lib/solaar/ui/window.py

```diff
@@ -578,16 +562,15 @@ def _update_receiver_panel(receiver, panel, buttons, full=False):
 	assert receiver
 
 	devices_count = len(receiver)
-	if receiver.max_devices > 1:
-		if devices_count == 0:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[0] % receiver.max_devices)
-		else:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[1] % (devices_count, receiver.max_devices))
-	else:
-		if devices_count == 0:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[0])
-		else:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[1])
+
+	paired_text = _('No device paired.') if devices_count == 0 else ngettext('%(count)s paired device.', '%(count)s paired devices.', devices_count) % { 'count': devices_count }
+
+	if(receiver.max_devices > 0):
+		paired_text += ngettext('\n\n<small>Up to %(max_count)s device can be paired to this receiver.</small>', '\n\n<small>Up to %(max_count)s devices can be paired to this receiver.</small>', receiver.max_devices) % { 'max_count': receiver.max_devices }
+	elif(devices_count > 0):
+		paired_text += _('\n\n<small>Only one device can be paired to this receiver.</small>')
```

Comment:
Would it be possible to use `"\n\n<small>%s</small" % _(...)` here to avoid HTML in translations? This would reduce the probability of messing it up in translations.

(here and above in the "Up to ..")